### PR TITLE
Replace interpolate_idents with stable code

### PR DESCRIPTION
**Pull request by dtolnay**
https://github.com/dtolnay/paste

### CODE REVIEW

File: src/api/from/from_array.rs [hunk 1]
```diff
@@ -89,15 +89,15 @@ macro_rules! impl_from_array {
                         let array_from_vec = <[$elem_ty; $elem_count]>::from(vec);
                         // FIXME: Workaround for arrays with more than 32 elements.
                         for i in 0..$elem_count {
-                            assert_eq!(array_from_vec[[i]], array[[i]]);
+                            assert_eq!(array_from_vec[i], array[i]);
```
Comment (gnzlbg) [inline src/api/from/from_array.rs]: This is so great! Thank you!

**Comment by gnzlbg** [comment]
@dtolnay It looks like some `fmt` tests are failing because `stringify!($id)` expands to`"$id "` instead of just `"$id"`.

**Comment by dtolnay** [comment]
I believe this was a compiler bug: https://github.com/rust-lang/rust/issues/55658. I implemented a workaround in paste 0.1.3.

**Comment by gnzlbg** [comment]
Thank you so so much, this is so awesome!

**Pull request merged**